pkg/scaffold/internal/machinery/errors.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machinery
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+// This file contains the errors returned by the scaffolding machinery
+// They are not exported as they should not be created outside of this package
+// Exported functions are provided to check which kind of error was returned
+
```

Comment:
Should these errors implement the `Unwrap` method?

Reply:
They do not wrap any error.